src/NzbDrone.Api/Frontend/Mappers/StaticResourceMapper.cs

```diff
@@ -27,6 +27,7 @@ public override bool CanHandle(string resourceUrl)
         {
             return resourceUrl.StartsWith("/Content") ||
                    resourceUrl.EndsWith(".js") ||
+                   resourceUrl.EndsWith(".map") ||
```

Comment:
source maps are in a different file, so we can have them included in the release builds as well, they don't get downloaded unless dev tools is open.